AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/Data/SignOutEventData.swift

```diff
@@ -11,12 +11,18 @@ struct SignOutEventData {
 
     let globalSignOut: Bool
 
+    #if os(iOS) || os(macOS) || os(visionOS)
     let presentationAnchor: AuthUIPresentationAnchor?
 
     init(globalSignOut: Bool, presentationAnchor: AuthUIPresentationAnchor? = nil) {
         self.globalSignOut = globalSignOut
         self.presentationAnchor = presentationAnchor
     }
+    #else
+    init(globalSignOut: Bool) {
```

Comment:
Is this needed? `watchOS` and `tvOS` do have an empty `AuthUIPresentationAnchor` that is used as placeholder